src/components/Location.js

```diff
@@ -0,0 +1,31 @@
+import defaultLocationImg from '../assets/default-location.svg';
+
+export const Location = ({ location }) => {
+  const { pageid, title, description, thumbnail, coordinates } = location;
+  const dist = typeof coordinates[0] === 'undefined' ? 0 : coordinates[0].dist;
+  const imgSrc =
+    typeof thumbnail === 'undefined' ? defaultLocationImg : thumbnail.source;
+
+  return (
+    <li className="location">
+      <div className="location__info">
+        <figure className="location__cover">
+          <img className="location__image" src={imgSrc} alt={title} />
```

Comment:
Are we guaranteed always to have a `title` value? What comes through if that’s not set? Does it show up as `alt=""`?

Reply:
Random aside, but your `className`s are also excellent.